content/en/blog/_posts/2023-09-15-userns-stateful-pods/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kuberntes 1.28: Alpha support to run stateful pods too!"
+date: 2023-09-15
+slug: userns-alpha
+---
+
+**Authors:** Rodrigo Campos Catelin (Microsoft), Giuseppe Scrivano (Red Hat), Sascha Grunet (Red Hat)
+
+Kubernetes v1.25 introduced support for user namespaces for only stateless
+pods. In Kubernetes 1.28 we lifted that restriction, after some design changes
```

Comment:
Does “we” mean Rodrigo Campos Catelin, Giuseppe Scrivano and Sascha Grunet? If not, reword this.

Reply:
Fixed